src/packages/shared-utils/package-check.ts

```diff
@@ -11,13 +12,8 @@ const secondClockStatuses = [
   SEATOOL_STATUS.PENDING_CONCURRENCE,
 ];
 
-const checkAuthority = (
-  authority: Authority | null,
-  validAuthorities: Authority[],
-) =>
-  !authority
-    ? false
-    : validAuthorities.includes(authority.toLowerCase() as Authority);
```

Comment:
Removing `.toLowerCase` means I need to update the test arguments to be normal case